### PR TITLE
200 and 400 added

### PR DESCRIPTION
**Pull request by SamR232**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -49,30 +49,70 @@ describe("GET /api/articles/:article_id", () => {
...
+  });
+  test("Status: 400 - invalid data type for votes", async () => {
+    let votesObj = { inc_votes: "dog" };
+    const { body } = await request(app)
+      .patch("/api/articles/1")
+      .send(votesObj)
+      .expect(400);
+    expect(body.msg).toEqual("Bad Request");
+  });
+});
```
Comment (HeatherEver) [inline __tests__/app.test.js]: Would be good to add more error tests for the PATCH request, think about incorrect article_id or incorrect path entered 👍

**Review by HeatherEver — CHANGES_REQUESTED** [review]
Code is looking really good ⭐  just add a couple more error tests for the PATCH 😄

File: __tests__/app.test.js (hunk 1 above)
Comment (SamR232) [reply __tests__/app.test.js]: updated with extra errors :)

**Review by HeatherEver — APPROVED** [review]
All good ⭐

**Pull request merged**